validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -213,13 +214,13 @@ public void subscribeToPersistentSubnets(Set<SubnetSubscription> subnetSubscript
 
   @Override
   public void sendSignedAttestation(final Attestation attestation) {
-    attestationPool.add(attestation);
+    attestationPool.add(ValidateableAttestation.fromSingle(attestation));
     eventBus.post(attestation);
   }
 
   @Override
   public void sendAggregateAndProof(final SignedAggregateAndProof aggregateAndProof) {
-    attestationPool.add(aggregateAndProof.getMessage().getAggregate());
+    attestationPool.add(ValidateableAttestation.fromAggregate(aggregateAndProof));
     eventBus.post(aggregateAndProof);
```

Comment:
I think we'll need to start sending these attestations to the `AttestationManager` since it won't pick it up from the eventBus anymore.  That's really what we want anyway design-wise.

Reply:
We still use `EventBus` at `GossipManagers` to take these attestations and put them on the wire. We could of course change that, but `AttestationManager` does not currently have a way of communicating downstream. On a different note, to add to your point, we have to consider our own attestations for our fork choice processing too, and sending them to `AttestationManager` would help fix that.

Reply:
Sounds like we need to have a way of sending some attestations downstream from `AttestationManager`. This would be applicable both for network layer `SAVED_FOR_FUTURE` labeled attestations, and attestations that we generated ourselves.

Reply:
I've implemented passing ungossiped attestations downstream.